core/src/test/scala/unit/kafka/server/ReplicaFetcherThreadTest.scala

```diff
@@ -103,11 +103,12 @@ class ReplicaFetcherThreadTest {
                                          failedPartitions: FailedPartitions,
                                          replicaMgr: ReplicaManager,
                                          quota: ReplicaQuota,
-                                         leaderEndpointBlockingSend: BlockingSend): ReplicaFetcherThread = {
+                                         leaderEndpointBlockingSend: BlockingSend,
```

Comment:
Should we also add tests here to test the old and the new version of the fetch request?

Reply:
Updated the testBuildFetch.